ansible_collections/ds/ansible_ds/playbooks/dscreate.yml

```diff
@@ -1,47 +1,41 @@
-- name: Playbook to create a new instance if it does not alreadfy exist
+---
+- name: Playbook to create a DS instance
   hosts: localhost
+
+  vars:
+    # Note: all variables values have be set to be 
+    # Instance name
+    instance: "i1"
+
+    # Non secure port (could be 0 if there is no such port)
+    port: 38901
+
+    # Secure port (could be 0 if there is no such port)
+    secure_port: 63601
+
+    # default Directory manager password
+    # here it is inherited from calling test playbook by it should be set by using:
+    #  echo "password" | ansible-vault encrypt_string --stdin-name rootpw
+    #rootpw: !vault |
+    #     $ANSIBLE_VAULT;1.1;AES256
+    #     65386361623432656133343230333636626164353230623935616632636361356265623530366232
+    #     6662333634333066613430616461623433633766333936330a616239636266666465343235316666
+    #     34373235366537323163623339373332323338326266356339303337363736646563623862366261
+    #     3435303734383335390a323638313539333338313866636134393731306162343837393966333735
+    #     6134
+
+    # Backend map using the backend name: suffix notation
+    # ( The dscreate.yml playbook expect at list one backend )
+    backends:
+        userroot: "dc=example,dc=com"
+        second: "dc=another example,dc=com"
+        peoplesubsuffix: "o=people,dc=example,dc=com"
+
+
   collections:
     - ds.ansible_ds
 
   roles:
-  - role: ansible_collections/ds/ansible_ds/playbooks/roles/install_dspkg
+  #- role: ansible_collections/ds/ansible_ds/playbooks/dsserver
+  - role: dsserver
```

Comment:
Please, add README.md for the role, it's a hard requirement for most of the Ansible automation. You can do this similar to this:
https://github.com/389ds/ansible-ds/blob/main/ansible_collections/ds/ansible_ds/playbooks/roles/dsbackup/README.md